### PR TITLE
Improve contributing docs

### PR DESCRIPTION
**Pull request by klaases**

### CODE REVIEW

**Comment by medyagh** [comment]
I signed it

**Comment by klaases** [comment]
I signed it

File: site/content/en/docs/contrib/guide.en.md [hunk 1]
```diff
@@ -1,52 +1,53 @@
 ---
 title: "Contributor Guide"
 linkTitle: "Guide"
-date: 2019-07-31
+date: 2019-11-09
```
Comment (medyagh) [inline site/content/en/docs/contrib/guide.en.md:4]: i m not sure what is this date is used for in the template,
but if we are updating it, it should be "2021"

Comment (klaases) [reply site/content/en/docs/contrib/guide.en.md]: I have updated the date with 2021, and committed the change.

**Review by sharifelgamal — APPROVED** [review]

**Pull request merged**